pkg/apis/scope/v0alpha1/types.go

```diff
@@ -62,8 +62,8 @@ type ScopeDashboardBinding struct {
 }
 
 type ScopeDashboardBindingSpec struct {
```

Comment:
> @bergquist do you think it would work to change this to store a reference to the parent scope(s) on the dashboard?

There should be no upper limit on how many scopes an dashboard can belong to. The same is true for the other way around. A scope should be able to have an unlimited amount of dashboards. (All thou its not gonna be practical to have more than a few hundred dashboards per scope).

Reply:
> no upper limit on how many scopes an dashboard can belong to

👍 makes sense, thanks for the additional context